app/Console/Commands/UpdateArchivedStreamsCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Facades\Youtube;
+use App\Models\Stream;
+use App\Services\Youtube\StreamData;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class UpdateArchivedStreamsCommand extends Command
+{
+    protected $signature = 'larastreamers:update-archived-streams';
+
+    protected $description = 'Update latest 50 archived (& live) streams';
```

Comment:
I like the idea to only check the currently live streams in the dedicated command, so in this command, we can focus on the finished streams. This way we can check finished streams at the end of every day.

PS: Currently finished streams will directly go to the archive, but after this PR I change this so they stay on the home page till the next day and we mark them as finished. This way we don't have to update the finished streams right away.

Reply:
But i do still think we need to include the streams with the `live` status because of issue #50. (Sometimes a stream get's set to private before the `CheckIfLiveStreamsHaveEndedCommand` has run and then they don't get updated.)

Reply:
Ok, good point. Then let us add the live streams to that command as well, since we don't have a lot of current leave streams. Can you add that?

Reply:
Yes, is included.